core/package.json

```diff
@@ -8,12 +8,12 @@
   "scripts": {
     "preinstall": "npx only-allow pnpm",
     "postinstall": "./init.sh",
-    "build": "turbo run build",
-    "test": "turbo run test",
-    "lint": "turbo run lint",
-    "format": "turbo run format",
-    "size": "turbo run size",
-    "coverage": "turbo run coverage",
+    "build": "turbo run build --filter '!./packages/contracts/lib/*'",
+    "test": "turbo run test --filter '!./packages/contracts/lib/*'",
+    "lint": "turbo run lint --filter '!./packages/contracts/lib/*'",
+    "format": "turbo run format --filter '!./packages/contracts/lib/*'",
+    "size": "turbo run size --filter '!./packages/contracts/lib/*'",
```

Comment:
If we remove hardhat thoroughly from our stack, we need also remove `size` entry script here which depends on hardhat which is used to calculate contract size to avoid exceeds limit.

Reply:
Fixed in latest commit.